src/sc/sidechain.cpp

```diff
@@ -352,4 +415,40 @@ bool Sidechain::checkCertCustomFields(const CSidechain& sidechain, const CScCert
     }
     return true;
 }
+
+bool CSidechain::GetCeasedCumTreeHashes(CFieldElement& lastEpochEndBlockCum, CFieldElement& ceasedBlockCum) const
```

Comment:
No need to return lastEpochEndBlockCum. Remove

Reply:
Done